lib/usecasing/base.rb

```diff
@@ -31,6 +40,13 @@ def perform(ctx = { })
 
       private
 
+      def concat_superclass_variable(variable)
```

Comment:
This method does not do any actual concatenation. Please find a better name to describe the action.

Reply:
Do you think merge_superclass_variables is a better name?